src/types/types.ts

```diff
@@ -83,6 +83,25 @@ export type FilteredResults = {
   isFiltered: boolean;
 };
 
+export type PlatformOS =
```

Comment:
I'd remove the type from here, as it's used only once there's no reason to have an exported type. It's used only as a tool in platform.ts. So instead I'd do this in platform.ts:
```ts
type PlatformOS = keyof typeof osMapping;
```
So that we don't have to maintain the data in both places.